apps/meteor/app/lib/server/functions/cleanRoomHistory.ts

```diff
@@ -83,7 +84,17 @@ export async function cleanRoomHistory({
 		});
 
 		if (threads.size > 0) {
-			await Subscriptions.removeUnreadThreadsByRoomId(rid, [...threads]);
+			const subscriptionIds: string[] = (
+				await Subscriptions.findUnreadThreadsByRoomId(rid, [...threads], { projection: { _id: 1 } }).toArray()
+			).map(({ _id }: { _id: string }) => _id);
+
+			const removedUnreadThreadsResponse = await Subscriptions.removeUnreadThreadsByRoomId(rid, [...threads]);
+
+			if (removedUnreadThreadsResponse.modifiedCount) {
+				for await (const id of subscriptionIds) {
+					await notifyOnSubscriptionChangedById(id);
```

Comment:
Most of other calls are `void` promises, is there a reason to await for them here?